tests/lib/nested.sh

```diff
@@ -690,6 +690,12 @@ defaults:
     journal:
       persistent: true
 EOF
+
+                        # add debugging kernel command line parameters
+                        for cmd in "dangerous" "systemd.journald.forward_to_console=1" "rd.systemd.journald.forward_to_console=1"; do
```

Comment:
there's also snapd.debug=1 which should enable debugs right from the boot

Reply:
I decided to leave this out because in some places once snapd.debug=1 is set, there is no way to turn it off, and so some tests would need to be adjusted for this, so instead just leaving it out for now I think is okay. When we eventually gain support for the env var SNAPD_DEBUG (if set) overriding the value of snapd.debug then I think we can do this by default